src/openklant/management/commands/migrate_to_v2.py

```diff
@@ -0,0 +1,475 @@
+
+        if digitaal_adres:
+            _data = openklant_client.create(
+                digitaal_addressen_path, digitaal_adres.dict()
+            )
+
+            if not isinstance(_data, dict):
+                logger.error(
+                    f"Unknown data received creating a digitaal adres: {_data}. "
+                    "Skipping klant."
+                )
+                continue
+
+            digitaal_adres_ref = _data.get("uuid")
+
+        if klant.subject and not klant.subject_identificatie:
+            subject_identificatie = generic_client.retrieve(klant.subject)
+
+            if not isinstance(subject_identificatie, dict):
+                logger.error(
+                    f"Unknown data received receiving a subject: {subject_identificatie}. "
+                    "Skipping klant."
+                )
+
+            klant.subject_identificatie = subject_identificatie
+
+        partij = klant.to_partij(digitaal_adres=digitaal_adres_ref)
+
+        if not partij:
+            logger.error("Unable to create partij, skipping klant..")
+            continue
+
+        response_data = openklant_client.create(partijen_path, partij.dict())
+
+        if response_data and "url" in response_data:
+            created_klanten.append(response_data["url"])
+
+    return created_klanten
+
+
+def _generate_dummy_token() -> str:
+    token_auth, _ = TokenAuth.objects.get_or_create(application="Migration application")
+    return token_auth.token
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser: CommandParser) -> None:
+        parser.add_argument(
+            "v1_url",
+            type=str,
+            metavar="https://example.openklant.nl",
+            help="URL of the Klanten API",
+        )
+
+        parser.add_argument(
+            "v2_url",
+            type=str,
+            metavar="https://example.klantinteracties.nl",
+            help="URL of the Klantinteracties API",
+        )
+
+    def handle(self, *args: Any, **options: Any) -> str | None:
+        access_token = os.getenv("ACCESS_TOKEN")
+        v1_url = options["v1_url"]
+        v2_url = options["v2_url"]
+
+        url_validator = URLValidator(schemes=["http", "https"])
+
+        for url in (v1_url, v2_url):
+            try:
+                url_validator(url)
+            except ValidationError as e:
+                return (
+                    f"Invalid URL(s) detected: {str(e.message)}. See "
+                    "migrate_to_v2 --help for correct usage."
+                )
+
+        next_url: str | None = ""
+
+        if not access_token:
+            raise ImproperlyConfigured("An access token is required to acces V1")
+
+        results = []
+
+        while next_url is not None:
+            klanten, next_url = _retrieve_klanten(
+                v1_url if next_url == "" else next_url, access_token
+            )
+            results.extend(_save_klanten(v2_url, klanten))
+
+        return "\n".join(results)
```

Comment:
I see this returns the results, but since it's a `Command`, is this also part of the output you see when running the command?